lib/Doctrine/ODM/PHPCR/Mapping/Annotations/Binary.php

```diff
@@ -21,9 +21,13 @@
 
 use Doctrine\Common\Annotations\Annotation;
 
+@trigger_error('The @Binary annotation is deprecated, use @Field(type="binary") instead.', E_USER_DEPRECATED);
```

Comment:
It's not about the Symfony version you use, it's about the addition of `@` in front of `trigger_error` (whereas Symfony 2.7-dev didn't have this `@` prefix). This means the error is silenced, unless you have a specific error handler for this type of errors.

Anyway, I'll comment this out (I want to have this PR merged asap).